tests/phpunit/tests/ProjectLocator.php

```diff
@@ -7,12 +7,33 @@
 
 namespace Required\Traduttore\Tests;
 
-use \WP_UnitTestCase;
+use \GP_UnitTestCase;
 use \Required\Traduttore\ProjectLocator as PL;
 
 /**
  *  Test cases for \Required\Traduttore\ProjectLocator.
  */
-class ProjectLocator extends WP_UnitTestCase {
+class ProjectLocator extends GP_UnitTestCase {
+	/**
+	 * @var \GP_Project
+	 */
+	public $root;
 
+	/**
+	 * @var \GP_Project
+	 */
+	public $sub;
+
+	/**
+	 * @var \GP_Project
+	 */
+	public $subsub;
+
+	public function setUp() {
+		parent::setUp();
```

Comment:
`GP_UnitTestCase` does access `GP_Locales` in its `setUp` method and somehow `GP_Locales` isn't loaded yet at this point. Even though I'm loading GlotPress in `bootstrap.php` using `require_once $_gp_tests_dir . '/includes/loader.php';`. I guess that's too late?

@ocean90 Any idea?

Reply:
Same issue when moving it right after `require $_tests_dir . '/includes/bootstrap.php';`.

It used to work in glotpress-json: https://github.com/swissspidy/glotpress-json/blob/master/tests/phpunit/bootstrap.php

Reply:
Can you try adding this https://github.com/GlotPress/GlotPress-WP/blob/develop/tests/phpunit/bootstrap.php#L21-L22 so https://github.com/GlotPress/GlotPress-WP/blob/develop/glotpress.php#L142-L152 doesn't fail.

Reply:
🤦‍♀️ I left that out on purpose because I thought the test case will set the permalink structure anyway:

https://github.com/GlotPress/GlotPress-WP/blob/43bb5383e114835b09fc47c727d06e6d3ca8114e/tests/phpunit/lib/testcase.php#L22-L25

By the way, the comment in https://github.com/GlotPress/GlotPress-WP/blob/develop/glotpress.php#L142-L152 could deserve some fixes. The last sentence is...weird.